k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageViewFragment.java

```diff
@@ -165,7 +166,7 @@ public void onDestroy() {
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
-            Bundle savedInstanceState) {
+                             Bundle savedInstanceState) {
```

Comment:
Well, the `settings.jar` given [here](https://github.com/k9mail/k-9/wiki/CodeStyle) broke the IDE 😅 . I am running Studio version 2.2.1. For the time being, I have manually corrected the indentation.